src/frontend/js/widgets/Dashboard/utils/learnerRoutesPaths.tsx

```diff
@@ -0,0 +1,70 @@
+import { defineMessages } from 'react-intl';
+
+/**
+ * All existing dashboard paths.
+ */
+export enum LearnerDashboardPaths {
```

Comment:
You renamed `TeacherDashboardPaths` into `TeacherRoutesPaths` but keep `LearnerDashboardPaths` as is.
Choose one and use it for both ?

Reply:
Good catch! I choose to keep `TeacherDashboardPaths` because "path" and "route" are quite synonyms.

Reply:
dashboard or routes doesn't matter, but it's nice to have the same naming both ways.